src/root/routes.tsx

```diff
@@ -98,6 +101,10 @@ const SiteRoutes = (): JSX.Element => {
               </RouteWithLoader>
             }
           />
+          <Route
+            path={getRouteById(AppRoutes.FAVOURITES)}
+            element={<FavouritesPage />}
```

Comment:
This is still fine for purposes of this pull request, but N.B. for when starting work on the API-enabled parts of this functionality: now that the login branch has been merged, you'll also want to wrap this up in a `RouteWithLoader` like the above routes to make sure the API key is properly loaded first before the route gets rendered and any data fetched.